ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -333,8 +333,9 @@ export default defineComponent({
           !!location.landDistrict || !!location.plan || !!location.exceptionPlan || !!location.reserveNumber
       }),
       landOwnershipLabel: computed(() => {
+        if (getMhrRegistrationOwnLand.value !== true || getMhrRegistrationOwnLand.value !== false) return '(Not Entered)'
```

Comment:
Could we just simply check for `null` value?

Reply:
Cameron thought I may need to check 'undefined' based on his comment above, so I changed 'check null' to 'check non true nor false'.

Reply:
How about `if (!getMhrRegistrationOwnLand.value) `? This would check for `null` and `undefined`.